src/concord-once.c

```diff
@@ -66,6 +86,75 @@ ccord_global_init()
     return CCORD_OK;
 }
 
+static void *
+malloc_fn_system(size_t length)
```

Comment:
`_ccord_` prefix for consistency

Reply:
What would be the purpose of that? It's `static` and only used in this TU. I see no reason to namespace it.

Reply:
The main idea here is prefixing with `_`, whether its `_mem` or `_ccord`

> It's static and only used in this TU.
1. We can quickly infer if a function is static at a glance by simply checking if it's prefixed by a `_` (no need to check its declaration)
2. That's how it is in the overall codebase, it's good to keep things consistent even if only for consistency's sake

Reply:
Alright.